docs/datafeed.md

```diff
@@ -0,0 +1,121 @@
+# Datafeed
+
+The datafeed service is a service used for handling the [_Real Time Events_](https://developers.symphony.com/restapi/docs/real-time-events). 
+For each interaction of a user within the IM, MIM or Room chat, an event will be sent to the datafeed.
+The bot can create a datafeed, list all created datafeeds or retrieve all the Real Time events within a datafeed through datafeed API.
+The datafeed service is a core service built on top of the Datafeed API and provide a dedicated contract to bot developers to work with datafeed. 
+
+The datafeed service is also the service where the [Activity API](../docs/activity-api.md) is built on top of.
+
+## How to use
+The central component for the contract between bot developers and  the Datafeed API is the `DatafeedService`.
+This service is accessible from the `SymphonyBdk` object by calling the `datafeed()` method.
+For instance:
+
+```java
+public class Example {
+
+    public static void main(String[] args) { 
+        // create bdk entry point
+        final SymphonyBdk bdk = new SymphonyBdk(loadFromClasspath("/config.yaml"));
+        
+        // create listener to be subscribed
+        final RealTimeEventListener listener = new RealTimeEventListener() {
+            @Override
+            public void onMessageSent(V4Initiator initiator, V4MessageSent event) {
+                log.info("Message sent");
+            }
+        };
+
+        // subscribe a listener
+        bdk.datafeed().subscribe(listener);
+
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            log.info("Stopping Datafeed...");
+            bdk.datafeed().stop();
+        }));
```

Comment:
Just wondering if we should keep it in this doc, at least in the main example

Reply:
I guess in the main example, only starting the datafeed service is enough?